libs/common-components/src/lib/top-toolbar/top-toolbar.component.html

```diff
@@ -1,7 +1,11 @@
 <div class="top-toolbar">
-  <gd-button id="left" [icon]="'arrow-left'" [tooltip]="'Scroll left'" (click)="moveLeft()"
-             *ngIf="showScroll && showLeft"></gd-button>
-  <ng-content></ng-content>
-  <gd-button id="right" [icon]="'arrow-right'" [tooltip]="'Scroll right'" (click)="moveRight()"
-             *ngIf="showScroll && showRight"></gd-button>
+  <gd-button [className]="'arrow-button'" class="arrow-left" id="left" [icon]="'arrow-left'" [tooltip]="'Scroll left'"
```

Comment:
Instead of `[icon]="'arrow-left'"` use `[icon]="'caret-left'"`. Same applies to right arrow